### DISCUSSION TITLE
Readability of the chat could be improved

### DISCUSSION THREAD

**Discussion by diomonogatari** [community/community#58646] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

Hey there,

I've been using VS Copilot Chat, almost daily now and I've come to the realization that it's readability is somewhat harsh.
Being the most predominant issue the line spacing.
Example:
![readibility example](https://github.com/community/community/assets/17832206/fae310bf-5c13-4b2b-8444-7cffec6afa6d)

Since its Options are still quite empty, I guess it would be good to consider the possibility of improving it via configuration params.
![suggestion](https://github.com/community/community/assets/17832206/6080f03f-1c39-4275-9bc2-d7de466d4640)

Good job with the tool :)

Cheers,
Dio

**Comment by Akash1134** [comment, marked as answer]
Hi @diomonogatari ,

Thank you for sharing the feedback. We will pass it on to the product team.
